src/index.js

```diff
@@ -0,0 +1,162 @@
+const tempDisplay = document.getElementById('tempValue');
+const increaseTemp = document.getElementById('increaseTempControl');
+const decreaseTemp = document.getElementById('decreaseTempControl');
+const landscape = document.getElementById('landscape');
+const cityNameDisplay = document.getElementById('headerCityName');
+const cityNameInput = document.getElementById('cityNameInput')
+const tempButton = document.getElementById('currentTempButton');
+const skySelect = document.getElementById('skySelect');
+const gardenContent = document.getElementById('gardenContent');
+const skyDisplay = document.getElementById('sky');
+const resetButton = document.getElementById('cityNameReset');
+const defaultCity = 'Seattle';
+
+let temperature = 88;
+tempDisplay.innerText = temperature;
+
+// Color of the temperature value number
+const updateTempColor = () => {
+tempDisplay.classList.remove('red', 'orange', 'yellow', 'green', 'teal');
+
+if (temperature >= 80) {
+    tempDisplay.classList.add('red');
+} else if (temperature >= 70) {
+    tempDisplay.classList.add('orange');
+} else if (temperature >= 60) {
+    tempDisplay.classList.add('yellow');
+} else if (temperature >= 50) {
+    tempDisplay.classList.add('green');
+} else {
+    tempDisplay.classList.add('teal');
+}
+};
+
+// Landscape emojis
+const updateLandscape = ()=>  {
+let landscapeDisplay;
+
+    if (temperature >= 80) {
+        landscapeDisplay = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    } else if (temperature >= 70) {
+        landscapeDisplay = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (temperature >= 60) {
+        landscapeDisplay = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃'; 
+    } else {
+        landscapeDisplay = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    }
```

Comment:
See comment above about incorporating a data structure to iterate over to get values instead of needing conditional logic.